### PR TITLE
Added support for NGPVAN ActionID OpenID

### PR DESCRIPTION
**Pull request by nickcatal**
Put together a full implementation of the NGPVAN [ActionID](http://developers.ngpvan.com/action-id) OpenID backend.

They'll offer OpenID Connect soon, but supporting their version of OpenID connect could involve some significant refactoring of the OpenID Connect implementation already in `python-social-auth`, so starting out with the standard OpenID 1.1 implementation would probably be a good first step.

Let me know what you think.

Thanks!

### CODE REVIEW

File: social/tests/models.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ class User(BaseModel):
     cache = {}
     _is_active = True
 
-    def __init__(self, username, email=None):
+    def __init__(self, username, email=None, **extra_user_fields):
```
Comment (nickcatal) [inline social/tests/models.py]: Added this here to better mimic what Django allows during the create_user step and thus better test `USER_FIELDS`

**Comment by nickcatal** [comment]
@omab let me know if you have any concerns here.

I need to incorporate this into a larger open source project and would prefer to simply have `pip install python-social-auth` as the setup step :)

**Comment by nickcatal** [comment]
@omab squashed this commit, but the addition of naver upstream causes a merge conflict (two "n" backends) you'll need to fix

FYI, this is currently live at https://public.ofaconnect.com/ if you want to check it out.

It'd be great to have this merged in so that I can change the requirements.txt file in ofa/connect to reference the pypi entry instead of a github fork.

**Comment by omab** [comment]
Took a couple months, but it's finally merged. Thanks!

**Comment by nickcatal** [comment]
Awesome!

@omab Can you push this to PyPI?

Thanks

**Pull request merged**